install/helm/superset/templates/configmap.yaml

```diff
@@ -24,4 +24,14 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
-{{ (.Files.Glob "config/*").AsConfig | indent 2 }}
+  {{- $defaultConf := .Files.Get "files/superset_config.py" |  nindent 4 }}
+  superset_config.py: |-
```

Comment:
This config should really be in a secret. Although the current config has nothing "secret" in it, if a user were to try and use this in a production-like environment, they would likely have secrets in their configs

Reply:
Also, pipe the outputs to `tpl` so that end users can use template syntax in their overrides

Reply:
Ok